subtrack.MAUI/Utilities/GetSubscriptionsFilter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using subtrack.DAL.Entities;
+using System.Linq.Expressions;
+
+namespace subtrack.MAUI.Utilities;
+
+public class GetSubscriptionsFilter
```

Comment:
GetSubscriptionsFilter will contain all our possible filterings like autopaid, overdue etc... So its more suitable as just a Dto with props that are mutable so the UI can use it